hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -11,12 +11,18 @@ tests:
       - series: 'kubevirt_hco_out_of_band_modifications_count{component_name="kubevirt/kubevirt-kubevirt-hyperconverged"}'
         # time:  0     1     2 3 4 5 6 7 8 9 10  11 12 13 14 15 16    17    18    19 20 21 22 23 24 25 26 27 28 29 30
         values: "stale stale 1 2 3 3 3 3 3 3 3   3  3  3  3  3  stale stale stale 1  1  1  1  1  1  2  2  2  2  3  3"
+      - series: 'kubevirt_hco_unsafe_modification_count{annotation_name="kubevirt.kubevirt.io/jsonpatch"}'
```

Comment:
kubevirt_hco_out_of_band_modifications_count is a counter. When there is no modification on operands, kubevirt_hco_out_of_band_modifications_count doesn't seen in the metrics, which means "stale" in the series above. When you do a change on any operand, it becomes "1". It cannot decrease back. You can only restart the operator and metric so it can become "stale" again. This series handles these  cases.

When it comes to kubevirt_hco_unsafe_modification_count, it is gauge. it may increase&decrease in time. As far as I see, it is zero when there is no json patch annotations. Therefore, I would adapt the series so that it can reflect all possible cases.